Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1848,4 +1874,24 @@ void TextEditor::set_visualize_trailing_whitespace(bool enabled)
     update();
 }
 
+void TextEditor::set_should_autocomplete_automatically(bool value)
+{
+    if (value == should_autocomplete_automatically())
+        return;
+
+    if (value) {
+        ASSERT(m_autocomplete_provider);
+        m_autocomplete_timer = add<Core::Timer>(m_automatic_autocomplete_delay_ms, nullptr);
+        m_autocomplete_timer->set_single_shot(true);
+        m_autocomplete_timer->stop();
+        m_autocomplete_timer->on_timeout = [this] {
+            try_show_autocomplete();
+        };
```

Comment:
```suggestion
    m_autocomplete_timer = Core::Timer::create_single_shot(m_automatic_autocomplete_delay_ms, [this] { try_show_autocomplete(); });
```

Reply:
That changed so incrementally I didn't even think of using that factory function...oops